### ISSUE TITLE
pod search Filament cannot find version 1.50.0

### ISSUE THREAD

**Issue by LibaoBambu** [google/filament#7524]
⚠️ **Issues not using this template will be systematically closed.**

**Describe the bug**
```[!] CocoaPods could not find compatible versions for pod "Filament":
  In Podfile:
    Filament (= 1.50.0)

None of your spec sources contain a spec satisfying the dependency: `Filament (= 1.50.0)`.
```

**Screenshots**
![20240125-160934](https://github.com/google/filament/assets/112543238/175cf485-64ee-4f28-b139-cee4699b37af)

**Comment by LibaoBambu** [comment]
`pod repo update --verbose` and `pod install --repo-update` not work for me

**Comment by xiamingwei-sudo** [comment]
i got sam problem,  when i use 1.49.0 in cocoapods.
hope a solution ,thanks

**Comment by bejado** [comment]
Sorry, we're having some problems with our CocoaPods publishing CI job. I just published version 1.50.0.
